ipatests/test_xmlrpc/test_host_plugin.py

```diff
@@ -337,6 +341,87 @@ def test_add_host_with_null_password(self, host):
         host.track_create()
         host.check_create(result)
 
+    @staticmethod
+    def modify_config_maxhostname(host_tracker, value):
+        command = host_tracker.make_command(
+            'config_mod',
+            **dict(
+                setattr=u'ipamaxhostnamelength={}'.format(value)))
+        command()
```

Comment:
command can raise an Exception if the configured value = the one you're trying to set (something similar to EmptyModlist). You need to take that into account otherwise some tests may fail.